docs/ruleset_functions/G311_exceptions/does_zone_meet_G3_1_1c.md

```diff
@@ -0,0 +1,65 @@
+# does_zone_meet_G3_1_1c
+
+**Schema Version:** 0.0.22
+
+**Description:** determines whether a given zone meets the G3_1_1c exception "If the baseline HVAC system type is 5, 6, 7, 8 use separate single-zone systems conforming with the requirements of system 3 or system 4 (depending on building heating source) for any spaces that have occupancy or process loads or schedules that differ significantly from the rest of the building. Peak thermal loads that differ by 10 Btu/h·ft2 (2.930710 W/sf) or more from the average of other spaces served by the system, or schedules that differ by more than 40 equivalent full-load hours per week from other spaces served by the system, are considered to differ significantly. Examples where this exception may be applicable include but are not limited to natatoriums and continually occupied security areas. This exception does not apply to computer rooms."
+"average" is not particularly well defined in this exception. 
+for the peak W/sf calculation I've assumed:
+	1. the zone being tested is not included
+ 	2. the average: (each space coincident peak W) / total_area
+
+**Inputs:** 
+- **B_RMD**
+- **zone**
+- **zones_and_systems** - this is a dict of the existing expected system types from the function `get_zone_target_baseline_system`
+
+**Returns:**  
+- **result**: an enum - either YES or NO
+ 
+**Function Call:**
+- **get_baseline_system_types**
+- **get_zone_peak_internal_load**
+- **get_list_hvac_systems_associated_with_zone**
+- **get_hvac_zone_list_w_area**
+- **find_zone**
+- **get_zone_eflh**
+- **zones_with_computer_room_dict_x**
+
+## Logic:
+- set the result variable to "No" - only a positive test can give it a different value: `result = NO`
+
+- first check if the system type is one of the following: `eligible_primary_system_types = ["SYS-5","SYS-6","SYS-7","SYS-8"]`
+- check if the starting_system_type is in the list of eligible system types: `if zones_and_systems[zone]["EXPECTED_SYSTEM_TYPE"] in eligible_primary_system_types:`
+	- create a list of zones that are on the same floor that have the same system type: `zones_on_same_system = []`
```

Comment:
I would determine function fo zone on same floor, because we use it in the vestibule RDS, to determine system-per-floor, and also here. Since we are not 100% confident about how to best determine it, it's better to isolate this logic into a function so that the update impacts all RDSs where this logic is used.

Reply:
created function [get_zones_on_same_floor](https://github.com/pnnl/ruleset-checking-tool/commit/69a426628062b59a732d86a18fe7456f8df52da0)